ChangeLog.md

```diff
@@ -27,6 +27,8 @@ See docs/process.md for more on how version tagging works.
   - Node: v10.19.0 -> v12.22.9
   - Chrome: v70 -> v74
   - Firefox: v55 -> v65
+  - Minimum Firefox version was further bumped up to Firefox 68 ESR - older
+  Firefox versions are not able to run the parallel browser harness (#25493)
```

Comment:
Maybe delete the line above (and use the same "Firefox: X -> Y" format as the above bumps)?